crates/bevy_color/src/color_ops.rs

```diff
@@ -0,0 +1,43 @@
+/// Methods for changing the luminance of a color. Note that these methods are not
+/// guaranteed to produce the same results in different color spaces, but are only
+/// approximations.
+pub trait Luminance: Sized {
+    /// Return the luminance of this color (0.0 - 1.0).
+    fn luminance(&self) -> f32;
+
+    /// Return a new version of this color with the given luminance.
```

Comment:
This should specify what happens if values outside of the range are supplied (and what the sensible range is).